spec/features/preassembly_gis_vector_accessioning_spec.rb

```diff
@@ -149,10 +150,10 @@
     expect_link_on_purl_page(druid:,
                              text: 'View in EarthWorks',
                              href: "#{Settings.earthworks_url}/stanford-#{bare_druid}")
-    expect_text_on_purl_page(druid:, text: 'This point shapefile represents all air monitoring stations active in ' \
-                                           'California from 2001 until 2003')
     expect(page).to have_no_text(object_label) # the original object label has been replaced
     expect(page).to have_text('Air Monitoring Stations: California, 2001-2003') # with the new object label
+    expect(page).to have_text('This point shapefile represents all air monitoring stations active in ' \
+                              'California from 2001 until 2003') # abstract
     expect(page).to have_text('cartographic') # type of resource
```

Comment:
we don't need to use `expect_text_on_purl_page` method, since this just makes an unnecessary reload of the PURL page